demo/admin/src/common/blocks/LinkListBlockWithAdditionalFields.tsx

```diff
@@ -0,0 +1,23 @@
+import { createListBlock } from "@comet/blocks-admin";
+import { TextLinkBlock } from "@src/common/blocks/TextLinkBlock";
+import { userGroupAdditionalItemFields } from "@src/userGroups/userGroupAdditionalItemFields";
+import { UserGroupChip } from "@src/userGroups/UserGroupChip";
+import { UserGroupContextMenuItem } from "@src/userGroups/UserGroupContextMenuItem";
+import { FormattedMessage } from "react-intl";
+
+export const LinkListBlockWithAdditionalFields = createListBlock({
```

Comment:
I wouldn't add a new link list block for this, keep it in the original block instead.

Reply:
This causes a lot of other problems because if kept in the original block, the LinkList is used in the Footer and there is then a type mismatch because of the addition of userGroup field.

Reply:
Please add `@ts-expect-error` and TODOs instead. Otherwise we'll never fix this. You could even create a bug task.